flaskr/config.py

```diff
@@ -0,0 +1,11 @@
+from hashlib import sha3_256
+
+
+class Config:
+    SECRET_KEY = 'powerful secretkey'
+    EMAIL_CONFIRMATION_EXPIRATION = 86400
+    PASSWORD_RESET_EXPIRATION = 86400
+    MESSAGE_SEPARATION_TOKEN = "[" + sha3_256(bytes(SECRET_KEY, "utf-8")).hexdigest() + "]"
+    EMAIL_CONFIRMATION_REQUIRED = False
+    BUDDY_LIMIT = 15
+    WEBSITE_PATH = "http://52.208.1.251:8080/"
```

Comment:
@stoyanbinev we just need to put the URL redirect over here, then its ready to merge into master for the release.

Reply:
Solved in #96